src/ILLink.CodeFix/DAMCodeFixProvider.cs

```diff
@@ -128,7 +139,18 @@ public override async Task RegisterCodeFixesAsync (CodeFixContext context)
 				generator.TypeExpression (attributeSymbol), attributeArguments)
 				.WithAdditionalAnnotations (Simplifier.Annotation, Simplifier.AddImportsAnnotation);
 
-			editor.AddAttribute (targetNode, attribute);
+			if (addAsReturnAttribute) {
+				// don't use AddReturnAttribute because it's the same as AddAttribute https://github.com/dotnet/roslyn/pull/63084
+				//editor.AddReturnAttribute (targetNode, attribute);
```

Comment:
I would just remove the commented-out code instead of leaving it commented out. It's easy to figure out how to convert back to `AddReturnAttribute` once a fix is available in all supported platforms.
```suggestion
```